src/system/Zikula/Module/SettingsModule/SettingsModuleVersion.php

```diff
@@ -30,7 +30,7 @@ public function getMetaData()
         $meta['description'] = $this->__('General site configuration interface.');
         //! module name that appears in URL
         $meta['url'] = $this->__('settings');
-        $meta['version'] = '2.9.9';
+        $meta['version'] = '3.0.0';
```

Comment:
probably `2.9.10` would be more appropriate. changing the first digit implies much larger changes.